### PR TITLE
2.5.0 release

### PR DESCRIPTION
**Pull request by daschuer**
This is the version bump PR to release 2.5.0 finally. 
We can do smoke testing before or after merge.

### CODE REVIEW

File: packaging/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+mixxx (2.4.2-1~focal) focal; urgency=medium
+
+  * Build of 2.4.2
+
+ -- RJ Skerry-Ryan <rryan@mixxx.org>  Tue, 26 Nov 2024 23:29:32 +0000
+
```
Comment (ronso0) [inline packaging/debian/changelog:6]: so this is actually missing in 2.4 branch?

Comment (daschuer) [reply packaging/debian/changelog:6]: This is auto-generated with a current time stamp during the Launchpad upload, but not committed to git. We need to do it manually at any time before a new release, it in the change-log.

Comment (ronso0) [reply packaging/debian/changelog:6]: Ah okay, thanks!
(I feel like I wondered about this for some previous release :laughing: hoepfully remember it this time)

**Comment by daschuer** [comment]
I will take the liberty and proceed with the release. Time is running out ...

**Pull request merged**